src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -837,34 +834,56 @@ public sealed class LpNormalizingEstimator : LpNormalizingEstimatorBase
         /// <param name="columns">Pairs of columns to run the normalization on.</param>
         /// <param name="normKind">Type of norm to use to normalize each sample.</param>
         /// <param name="substractMean">Subtract mean from each value before normalizing.</param>
-        public LpNormalizingEstimator(IHostEnvironment env, (string outputColumnName, string inputColumnName)[] columns,
+        internal LpNormalizingEstimator(IHostEnvironment env, (string outputColumnName, string inputColumnName)[] columns,
             NormalizerKind normKind = Defaults.NormKind, bool substractMean = Defaults.LpSubstractMean)
-             : this(env, columns.Select(x => new LpNormalizingTransformer.LpNormColumnInfo(x.outputColumnName, x.inputColumnName, substractMean, normKind)).ToArray())
+             : this(env, columns.Select(x => new LpNormColumnInfo(x.outputColumnName, x.inputColumnName, substractMean, normKind)).ToArray())
         {
         }
 
         /// <summary>
         /// Create a <see cref="LpNormalizingEstimator"/> that takes multiple pairs of columns.
         /// </summary>
-        public LpNormalizingEstimator(IHostEnvironment env, params LpNormalizingTransformer.LpNormColumnInfo[] columns)
+        internal LpNormalizingEstimator(IHostEnvironment env, params LpNormColumnInfo[] columns)
             : base(env, columns)
         {
         }
     }
 
     /// <summary>
-    /// Global contrast normalizing estimator allow you take columns and performs global constrast normalization on them.
+    /// Global contrast normalizing estimator takes columns and performs global constrast normalization.
```

Comment:
Please add a remarks section either here or the catalog that explains what `global contrast normalization` is :)

Reply:
This is an open task in the main doc issue, I won't be doing this as part of this PR. Thank you for pointing this out though.

---
In reply to: [254013342](https://github.com/dotnet/machinelearning/pull/2366#discussion_r254013342) [](ancestors = 254013342)